proto/v1/database_service.proto

```diff
@@ -241,6 +241,18 @@ message GetDatabaseMetadataRequest {
   // The name of the database to retrieve metadata.
   // Format: instances/{instance}/databases/{database}/metadata
   string name = 1 [(google.api.field_behavior) = REQUIRED];
+
+  enum MetadataView {
```

Comment:
I think we can just review the whole thing as we pretty much need the same view in all places.